include/mender-alloc.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file      mender-alloc.h
+ * @brief     Mender memory management functions
+ *
+ * Copyright Northern.tech AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __MENDER_ALLOC_H__
+#define __MENDER_ALLOC_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+#include <stddef.h> /* size_t */
+
+/**
+ * Standard parameters and semantics apply to the three types and functions below.
```

Comment:
```suggestion
 * Standard parameters and semantics apply to the types and functions below.
```

:thinking: The "three" there is what confused me in the first round. I am not a grammar expert but "three apples and pears translates in my head to "three apples and three pears"

Reply:
Easy fix then! :)